Marlin/language_de.h

```diff
@@ -91,73 +91,73 @@
 #define MSG_PID_D                           "PID D"
 #define MSG_PID_C                           "PID C"
 #define MSG_ACC                             "A"
-#define MSG_VXY_JERK                        "V xy Ruck"
-#define MSG_VZ_JERK                         "V z  Ruck"
-#define MSG_VE_JERK                         "V e  Ruck"
+#define MSG_VXY_JERK                        "V XY Jerk"
+#define MSG_VZ_JERK                         "V Z  Jerk"
+#define MSG_VE_JERK                         "V E  Jerk"
 #define MSG_VMAX                            "V max " // space by purpose
 #define MSG_X                               "X"
 #define MSG_Y                               "Y"
 #define MSG_Z                               "Z"
 #define MSG_E                               "E"
 #define MSG_VMIN                            "V min"
-#define MSG_VTRAV_MIN                       "VTrav min"
+#define MSG_VTRAV_MIN                       "V min Leerfahrt"
 #define MSG_AMAX                            "A max " // space by purpose
-#define MSG_A_RETRACT                       "A Rückzug"
-#define MSG_A_TRAVEL                        "A Reise"
-#define MSG_XSTEPS                          "X steps/mm"
-#define MSG_YSTEPS                          "Y steps/mm"
-#define MSG_ZSTEPS                          "Z steps/mm"
-#define MSG_ESTEPS                          "E steps/mm"
+#define MSG_A_RETRACT                       "A Retract"
+#define MSG_A_TRAVEL                        "A Leerfahrt"
+#define MSG_XSTEPS                          "X Steps/mm"
+#define MSG_YSTEPS                          "Y Steps/mm"
+#define MSG_ZSTEPS                          "Z Steps/mm"
+#define MSG_ESTEPS                          "E Steps/mm"
 #define MSG_TEMPERATURE                     "Temperatur"
 #define MSG_MOTION                          "Bewegung"
 #define MSG_VOLUMETRIC                      "Filament"
 #define MSG_VOLUMETRIC_ENABLED              "E in mm³"
-#define MSG_FILAMENT_DIAM                   "Filament D"
+#define MSG_FILAMENT_DIAM                   "Filamentdurchmesser"
```

Comment:
Scroll-Feature is already there for the boot-screen. Alone it needs a bit more - updates. Currently the screen in the menu is refreshed only for an event (encoder - click). That gives low load and immediate response. For a nice scrolling we'd need more then ~4 refreshes a second - ~50% of the processor time we have (for the GLCD's) As long as no program is running that may work, but not during a print.